main.go

```diff
@@ -74,6 +74,26 @@ func main() {
 	router.DELETE("/users/:id", pCtrl.DeleteUser)
 	// TODO: panics
 	router.GET("/users/:user_id", pCtrl.GetUserById)
+	router.POST("/exams", pCtrl.CreateExam)
+	router.POST("/exams/:id/edit", pCtrl.EditExam)
```

Comment:
`PATCH` statt `POST` und `edit` aus der URL rausnehmen. REST hat in der URL immer nur Objekte, keine Aktionen